reference/orders.v2.oas2.yml

```diff
@@ -1058,6 +1079,363 @@ paths:
       parameters:
         - $ref: '#/components/parameters/order_id_path'
         - $ref: '#/components/parameters/shipping_consignment_id'
+  '/orders/{order_id}/fees':
+    parameters:
+    get:
+      summary: Get Fees
+      tags:
+        - Order Fees
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/orderFees'
+              examples:
+                response:
+                  value:
+                    fees:
+                      - id: 1
+                        type: custom_fee
+                        display_name_customer: A Fee
+                        display_name_merchant: A Fee
+                        source: v2
+                        base_cost: '12.3000'
+                        cost_exc_tax: '12.3000'
+                        cost_inc_tax: '12.30000'
+                        cost_tax: '0.0000'
+                        tax_class_id: 22
+                      - id: 2
+                        type: custom_fee
+                        display_name_customer: B Fee
+                        display_name_merchant: B Fee
+                        source: v2
+                        base_cost: '4.0000'
+                        cost_ex_tax: '4.0000'
+                        cost_inc_tax: '12.0000'
+                        cost_tax: '8.0000'
+                        tax_class_id: 0
+        '404':
+          $ref: '#/components/responses/404_Resp'
+      operationId: getOrderFees
+      description: 'Get all fees for an order. '
+      parameters:
+        - $ref: '#/components/parameters/order_id_path'
+  '/orders/{order_id}?include=fees':
```

Comment:
this probably is not a new endpoint, we should update the existing one and update `include` param instead

Reply:
Thanks updated, I had added in the existing ones and created new ones as well to explain it explicitly for fees but lint check did not allow query params anyways